TAScheduler/AcceptanceTests/test_create_account.py

```diff
@@ -0,0 +1,97 @@
+from django.test import TestCase, Client
+
+from TAScheduler.models import User, Administrator
+
+
+class SuccessfulCreate(TestCase):
+    user = None
+    account = None
+    info = None
+
+    def setUp(self):
+        self.user = Client()
+        temp = User(email_address="testadmin@uwm.edu", password="pass", first_name="Test", last_name="Test",
+                    home_address="Random location", phone_number=9990009999)
+        temp.save()
+        self.account = Administrator(user=temp)
+        self.account.save()
+        self.info = {"email_address", "test@uwm.edu", "password", "pass", "first_name", "test", "last_name", "ignore",
+                     "home_address", "3400 N Maryland Ave", "phone_number", 4142292222, "role", "TA"}
+
+    # TODO need to make tests work with sessions but I am unsure of how to start that
+    def test_creation(self):
+        self.user.post("/home/manageaccount/create", self.info)
+        self.assertTrue(User.objects.filter(email_address="test@uwm.edu").count() is 1)
+
```

Comment:
Maybe another test: even though that object is in the database, does it contain all of the info we sent into it? 
Also, maybe another test for a .context["message"] = "successfully created account."